### DISCUSSION TITLE
delete the existing hubs instance in aws

### DISCUSSION THREAD

**Discussion by jsdude0220** [mozilla/hubs#3735] [Hubs Cloud Questions] [answered]
Hello,
I have one old hubs instance, but I can't use it because it was crashed.
so I would like to delete it and create the new hubs with old hubs domain.
what is the way to do this?
Please let me know who knows this.
Thanks in advance.

**Comment by jsdude0220** [comment]
Hi, any good news?

**Comment by robinkwilson** [comment, marked as answer]
Hello! Yes you can absolutely delete it! 

Do you need any data in it? Then follow this guide: https://hubs.mozilla.com/docs/hubs-cloud-aws-backup-and-restore.html

Otherwise, go to "AWS management console" > "Cloudformation" > Select the hubs cloud stack you want to delete (if no stacks are displayed, check your region in the upper right hand corner) > Click "Delete"

Then after the delete is finished, you can deploy a new hubs cloud on the same domain! (I do this all the time :) )

**Reply by robinkwilson** [reply]
Marking as answer, unmark if this doesn't answer your question!
